### ISSUE TITLE
BUG: showOffsetsSelected not working

### ISSUE THREAD

**Issue by glaprida** [GrapesJS/grapesjs#4998]
### GrapesJS version

- [X] I confirm to use the latest version of GrapesJS

### What browser are you using?

Chome v111

### Reproducible demo link

https://codepen.io/glaprida/pen/YzOJOGq

### Describe the bug

**How to reproduce the bug?**
1. Open the codepen link. Drag any basic block to canvas with some margin or padding.
2. You will see that `showOffsets` works as expected (when hovering an element that is NOT selected)
3. Selecting the element hides all offsets. It doesn't show the selected element offsets even if you hover it. (`showOffsetsSelected` is set to true)

**What is the expected behavior?**
To be able to see the offsets of the selected element.

**What is the current behavior?**
It does not show them

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by glaprida** [comment]
Hi, I see that now it works while hovering the element even if it is selected (previously it only worked on non-selected elements)

...but it only shows the offsets while hovering (I thought `showOffsetsSelected` would show the offsets of the selected element even if it wasn't being hovered)

Is this the intended behaviour?

Any suggestions for how I can see the offsets of the selected element when it's not being hovered?

Big thanks !

**Comment by artf** [comment]
Yeah at the moment this is what is intended to do. Probably one day we'll revisit this to make it more customizable (ability to extend/create style handlers on canvas)

**Comment by glaprida** [comment]
Hi there, sorry for the insistence... but is there any way you could think of that I can achieve this?

What I need is to show the offsets of the selected element, no matter if it's hovered or not.
